src/rust/ide/lib/view/src/debug_scenes/documentation_generator_view.rs

```diff
@@ -0,0 +1,80 @@
+#![allow(missing_docs)]
+
+use ensogl::traits::*;
+use ensogl::prelude::*;
+
+use ensogl::system::web;
+use ensogl::system::web::NodeInserter;
+use ensogl::display::symbol::DomSymbol;
+use web::StyleSetter;
+use ensogl::display::world::*;
+use ensogl::display::navigation::navigator::Navigator;
+
+use nalgebra::Vector2;
+use nalgebra::Vector3;
+use wasm_bindgen::prelude::*;
+
+use crate::graph_editor::component::visualization::MockDocGenerator;
+use crate::graph_editor::builtin::visualization::native::documentation::doc_style;
+
+fn generate_mock_doc() -> String {
+    let sample_data_gen = MockDocGenerator::default();
+    let default_input   = sample_data_gen.generate_data();
+    let program         = std::env::args().nth(1).unwrap_or(default_input);
```

Comment:
this needs good documentation. I dont see any description about the accepted cmd args to this scene. Please add it to the module doc.